src/OctoshiftCLI.Tests/GithubApiTests.cs

```diff
@@ -2552,6 +2552,60 @@ public async Task UpdateSecretScanningAlert_Calls_The_Right_Endpoint_With_Payloa
             _githubClientMock.Verify(m => m.PatchAsync(url, It.Is<object>(x => x.ToJson() == payload.ToJson()), null));
         }
 
+        [Fact]
+        public async Task GetEnterpriseServerVersion_Returns_Null_If_Not_Enterprise_Server()
+        {
+            // Arrange
+            var url = "https://api.github.com/meta";
+            var response = $@"
+            {{
+                ""verifiable_password_authentication"": true,
+                ""packages"": [
+                ],
+                ""dependabot"": [
+                ]
+            }}";
+
+
+            _githubClientMock
+                .Setup(m => m.GetAsync(url, null))
+                .ReturnsAsync(response);
+
+            // Act
+            var version = await _githubApi.GetEnterpriseServerVersion();
+
+            // Assert
+            version.Should().BeNull();
+        }
+
+        [Fact]
+        public async Task GetEnterpriseServerVersion_Returns_Version()
+        {
+            // Arrange
+            var url = "https://api.github.com/meta";
+            var response = $@"
+            {{
+                ""verifiable_password_authentication"": true,
+                ""packages"": [
+                ],
+                ""dependabot"": [
+                ],
+                ""installed_version"": ""3.7.0""
+            }}
+        ";
```

Comment:
```suggestion
            }}";
```

this was missed 😅

Reply:
the `dotnet format` didn't do anything for me 😞